### DISCUSSION TITLE
Wintercms' htacces avoid me to access to subdomain

### DISCUSSION THREAD

**Discussion by rabimx** [wintercms/winter#1109] [Q&A] [answered]
Hi,
I created a subdomain for install moodle, but the htaccess rules avoid me to access to the subdomain. Unfortunately, I don't know how can I set a rule that allow the subdomain to work. Spmebody can help me? Thanks in advance.

**Comment by mjauvin** [comment, marked as answer]
Try this:

https://octobercms.com/forum/post/ignore-specific-subfolders-via-htaccess

For future: 

> If you want October CMS to ignore specific sub-folder rather than returning a 404 then add the following rule in .htaccess right after RewriteEngine On:
RewriteRule ^(foobar1|foobar2)($|/) - [L]
Where, foobar1 and foobar2 are sub-folders.
Reload your Apache config and it should work.

**Reply by rabimx** [reply]
Thanks! It worked.
